twindle-cli/twitter/types.d.ts

```diff
@@ -0,0 +1,73 @@
+export interface Mention {
+  start: number;
+  end: number;
+  username: string;
+}
+
+export interface Hashtag {
+  start: number;
+  end: number;
+  hashtag: string;
+}
+
+export interface EntityUrl {
+  start: number;
+  end: number;
+  /** format: `https://t.co/[REST]` */
+  url: string;
+  expanded_url: string;
+  /** The possibly truncated URL */
+  display_url: string;
+  status: number;
+  title: string;
+  description: string;
+  unwound_url: string;
+  images?: [
+    {
+      url: string;
+      height: number;
+      width: string;
+    }
+  ];
+}
+
+export interface Attachment {
+  poll_id?: string[];
+  media_keys?: string[];
+}
+
+export interface ConversationsResponseData {
+  public_metrics: {
+    retweet_count: number;
+    reply_count: number;
+    like_count: number;
+    quote_count: number;
+  };
+  conversation_id: string;
+  id: string;
+  text: string;
+  author_id: string;
+  referenced_tweets: [
+    {
+      type: "retweeted" | "quoted" | "replied_to";
+      id: string;
+    }
+  ];
+  created_at: string;
+  entities?: {
+    mentions?: Mention[];
```

Comment:
If it is possible, you may move the type definitions into a `types` folder?

Reply:
Ok. But we would need to keep the types.js in the root at least, so it can be used globally

Reply:
Typescript 😂

Reply:
@proful Not really

These are here to assist the coding experience. No compilers or extensions needed. Anyone with VScode and Atom will benefit, out of box.

Reply:
This is fine but be careful of over usage.